paperqa/readers.py

```diff
@@ -39,7 +39,13 @@ def parse_pdf_to_pages(
                     f" {file.page_count} for the PDF at path {path}, likely this PDF"
                     " file is corrupt."
                 ) from exc
-            text = page.get_text("text", sort=True)
+
+            # Extract text blocks, which are already in the correct order, from the page
+            blocks = page.get_text("blocks", sort=False)
+
+            # Concatenate text blocks into a single string
+            text = "\n".join(block[4] for block in blocks if len(block) > 4)
```

Comment:
What do we lose with `sort=False`? I am wondering why we had `sort=True` originally (it predates my time at FutureHouse)

Reply:
If the problem isn't `sort=False`, can you revert to `sort=True` there? Just to keep diff smaller

Reply:
Using `sort=False` retains the correct order of blocks in two-column pdfs (as well as one-column pdfs).